apps/web/migrations/0001_mighty_newton_destine.sql

```diff
@@ -0,0 +1 @@
+ALTER TABLE `user` ADD `hasOnboarded` integer DEFAULT false;
```

Comment:
@Dhravya should this be a part of setup? should the meta files also be commited?

Reply:
Yes, this and the meta file both are a part of the commit 

this allows other contributors to run wrangler d1 migrations apply command